presto-product-tests/README.md

```diff
@@ -28,3 +28,121 @@ Product tests are not run by default. To start them use run following command:
 ```
 java -jar target/presto-product-tests-*-executable.jar --config-local file://`pwd`/tempto-configuration-local.yaml
 ```
+
+## Running tests with Using preconfigured docker based clusters
+
+In case you do not have an access to hadoop cluster or do not want to spent your time on configuration, this section
+describes how to run product tests with preconfigured docker based clusters.
+
+### Prerequisites
+
+* docker >= 1.5
+
+[https://docs.docker.com/installation/#installation](https://docs.docker.com/installation/#installation)
+
+For linux users:
+```
+wget -qO- https://get.docker.com/ | sh
+```
+
+For Mac OS X you need to install docker-machine.
+
+[https://docs.docker.com/engine/installation/mac/](https://docs.docker.com/engine/installation/mac/).
+
+* Python >= 2.6
+
+* Vagrant >= 1.8
+
+* Presto docker development environment tool
+
+```
+pip install presto-docker-devenv
+```
+
+### Running product tests
+
+Below manual describe how to set up and teardown docker clusters needed to run product tests. 
+It also covers actual product tests execution with usage of these clusters.
+
+All of the steps from 2 to 8 are covered by ```presto-product-tests/bin/run.sh```. 
+You can use this script instead manually going through these steps. 
+
+1. Create docker-machine (only for Mac OS X users)
+
+When using Mac OS X and docker-machine, you need to create a machine with 4GB memory limit. 
+It is required as this machine is going to run one node hadoop and three node presto cluster.
+
+```
+docker-machine create -d virtualbox --virtualbox-memory 4096 default
+```
+
+> Note when you are using Linux ```presto-devenv``` will bind hadoop and presto ports to localhost address.
+> Please ensure that 8080, 8088, 10000, 50070, 500075 are free.
+
+2. Start hadoop cluster
+
+```
+presto-devenv hadoop start
```

Comment:
We should probably add some kind of progress loging for that command. It takes time to download all the images, and withoug any logging it looks like it hangs.